components/card.js

```diff
@@ -0,0 +1,67 @@
+import styled from "styled-components";
+
+const CardImageContainer = styled.div`
+	background: radial-gradient(
+		${(props) => props.theme.colors.cardImageBackgroundGradient},
+		transparent
+	);
+
+	@media not all and (max-width: 600px) {
+		padding: 12px;
+		margin: -12px;
+
+		display: flex;
+		flex-direction: row;
+		align-items: center;
+	}
+
+	@media (max-width: 600px) {
+		display: flex;
+		justify-content: center;
+
+		align-self: center;
+		width: 100%;
+	}
+`;
+
+export const CardImage = styled.img`
+	display: block;
+	width: 96px;
+	image-rendering: ${(props) => props.pixelated && "pixelated"};
+`;
+
+function UnstyledCard({ src, pixelated, alt, className, children }) {
+	return (
+		<div className={className}>
```

Comment:
Rather than having a component that returns an unstyled div that then gets style applied to it, i would probably just style the div and use that in the component

Reply:
Fixed in 1f2d910c82647d3d7b13abb5930be3745fc121f1